HW02_A_Graph-Fails.Rmd

```diff
@@ -61,6 +80,31 @@ ggplot() +
 
 ```
 
+**The attribute *alpha* is a value between 0 and 1, and it sets opacity of the point. Setting it to 2 is meaningless. You can set the attribute *size* to make the point bigger.** 

```

Comment:
Yes, I observed the same thing. I wonder if R just ignores the alpha = 2 argument in this case?

Reply:
Not pretty sure. At least the two images seemed to be the same in this case.